### PR TITLE
Support vector

### LINKED ISSUE #303: Class to represent a vector in arbitrary space-time symbolically
🐞 **Problem**
A class, which is derived from Tensor can be used to represent vectors(1st Order Tensor) in a spacetime.

🎯 **Goal**

  - Create such a class in symbolic module

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by OVSofia**
Fixes #303

### CODE REVIEW

**Comment by OVSofia** [comment]
@ritzvik  I have created a new branch for this issue.

I've checked that TensorClass already support a vector in arbitrary space-time symbolically. The proof of this can be seen in test_tensor.py in test_tensor1D().
But if you want a specific VectorClass anyway, I've created a class specifically.

**Comment by ritzvik** [comment]
> @ritzvik I have created a new branch for this issue.
> 
> I've checked that TensorClass already support a vector in arbitrary space-time symbolically. The proof of this can be seen in test_tensor.py in test_tensor1D().
> But if you want a specific VectorClass anyway, I've created a class specifically.

Ya! Would see this in time.
@spino17 , why don't you take a look?

File: src/einsteinpy/tests/test_symbolic/test_vector.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import numpy as np
+from sympy import sin, symbols
+
+from einsteinpy.symbolic.vector import GenericVector
```
Comment (ritzvik) [inline src/einsteinpy/tests/test_symbolic/test_vector.py]: We'll see to the tests after the requested changes are made :)

File: docs/source/api/symbolic/vector.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Vector module
+=============
+
+This module contains the class GenericVector to represent a vector in arbitrary space-time symbolically
```
[the remark this answers is not in the document]
Comment (ritzvik) [reply docs/source/api/symbolic/vector.rst]: **Sorry**, there is not mistake , my bad
@OVSofia 🤣

**Pull request merged**